skatepark/block-templates/index.html

```diff
@@ -0,0 +1,32 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:query {"tagName":"main","queryId":1,"query":{"perPage":10,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":""}} -->
+<main class="wp-block-query">
+	<!-- wp:post-template -->
+	<!-- wp:group {"layout":{"inherit":true}} -->
+	<div class="wp-block-group">
+		<!-- wp:post-title {"isLink":true} /-->
+		<!-- wp:post-featured-image {"isLink":true} /-->
+		<!-- wp:post-excerpt /-->
+		<!-- wp:spacer {"height":40} -->
+		<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
+		<!-- /wp:spacer -->
+	</div>
+	<!-- /wp:group -->
+	<!-- /wp:post-template -->
+	<!-- wp:group {"layout":{"inherit":true}} -->
+		<div class="wp-block-group">
+		<!-- wp:query-pagination -->
+		<div class="wp-block-query-pagination"><!-- wp:query-pagination-previous {"label":"Previous Page"} /-->
+	
+			<!-- wp:query-pagination-numbers /-->
+	
+			<!-- wp:query-pagination-next {"label":"Next Page"} /--></div>
```

Comment:
I wonder if we still need the label attributes here since these labels are the default text, and Ben's Gutenberg PR which removed the arrows: https://github.com/WordPress/gutenberg/pull/33626

Reply:
We definitely should do that for Blockbase

Reply:
For blockbase: https://github.com/Automattic/themes/pull/4331